src/components/HomePage/Banner.js

```diff
@@ -224,10 +224,10 @@ function Banner() {
 							style={{ marginBottom: 10, fontWeight: 500 }}
 							component='h3'
 						>
-							<time dateTime={hothStart.toISOString()} hidden>
-								{month} {startDay}{endDayString}, 2022
+							<b>Date: </b><time dateTime={hothStart.toISOString()} >
+								{month} {startDay}{endDayString}, 2023
```

Comment:
We should not be hardcoding the year, but also getting the year from the hothStart date.